### PR TITLE
Try to stabilize testErrorHandlingFailedToSignal test

### PR DESCRIPTION
**Pull request by MarianMacik**
@sutaakar @baldimir @mswiderski Just a test to stabilize one (actually 6 with parameterized) test.

### CODE REVIEW

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java [hunk 1]
```diff
@@ -743,6 +743,13 @@ public void testGetProcessInstancesWithOrderByClause() throws Exception {
 
     @Test
     public void testErrorHandlingFailedToSignal() throws Exception {
+        List<ExecutionErrorInstance> errors = processAdminClient.getErrors(CONTAINER_ID, false, 0, 100);
+        if (!errors.isEmpty()) {
+            String[] ids = errors.stream().map(ExecutionErrorInstance::getErrorId).toArray(String[]::new);
+            processAdminClient.acknowledgeError(CONTAINER_ID, ids);
+            errors = processAdminClient.getErrors(CONTAINER_ID, false, 0, 10);
+            assertEquals(0, errors.size());
```
Comment (MarianMacik) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java]: @mswiderski the test has failed on this assert so the only explanation is that there are more than 100 errors to ack probably. I will try it with 1000.

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
I see it is green, is it ready to be merged?

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
Ready to be merged. I will also backport this to 7.7.x, right?

**Comment by mswiderski** [comment]
Yes, Please prepare backport to 7.7.x, thanks

**Pull request merged**